server/__test__/database/index.test.js

```diff
@@ -0,0 +1,13 @@
+const dbBuild = require('../../database/config/build');
+const connection = require('../../database/config/connection');
+const { getNews } = require('../../database/queries');
+
+beforeEach(() => dbBuild());
+afterAll(() => connection.end());
+
+test('should get news from the news table', async () => {
+  const { rows } = await getNews();
+  return expect(rows[5].content).toEqual(
+    'Researchers simulated a tailgating situation with a small group of overweight but healthy men and examined the impact of eating and drinking on their livers using blood tests and a liver scan.',
```

Comment:
If you wanted this test to pass even if the fake data's order changed, you could do something like this:
```
const allContent = rows.map(row => row.content);
expect(allContent).toContain('Researchers simulated a tailgating situation with a small group of overweight but healthy men and examined the impact of eating and drinking on their livers using blood tests and a liver scan.
```
For more on `toContain` you can look here: https://jestjs.io/docs/en/expect#tocontainitem
This might save you or others a little bit of time in the future, if the fake data gets rearranged or new items get added to it, then you won't have to find this failing test and figure out how to fix it :)

Reply:
Thank you, Tom.

Reply:
Afwan! Hopefully it helps.